cypress/integration/tasks.ts

```diff
@@ -1,7 +1,8 @@
 describe("App", function () {
   it("Contains a link to learn about React", function () {
     cy.visit("/")
-    cy.get(".App-header").contains("My Tasks")
+    cy.get(".AppHeader").contains("My Tasks")
     cy.get('[type="checkbox"]').should("exist")
+    // cy.findByTestId("kittykat")
```

Comment:
⛏️ It's good practice to remove comments unless they are explaining something in the code that can't be understood. And even in that case, it's a good chance to ask yourself - "if this can't be understood, is there a way to re-write the code so that it's clear without a comment?" (also, still love "kittykat" as the default 😹 )